host/src/server/api/admin.rs

```diff
@@ -0,0 +1,114 @@
+use axum::{extract::State, routing::post, Router};
+
+use crate::{interfaces::HostResult, ProverState};
+
+pub fn create_router() -> Router<ProverState> {
+    Router::new()
+        .route("/admin/pause", post(pause))
+        .route("/admin/unpause", post(unpause))
```

Comment:
pause on/off instead of 2 entries?
Also maybe need security token to avoid unexpected pause

Reply:
IMO 2-entries design is more more clear, and operator doesn't have to remember the parameter type.

Reply:
no problem, then please take security token into account.

Reply:
You mean "/v*/proof" and "/admin/" APIs use different JWT tokens?

Reply:
up 2 you, I remember we didn't force JWT in /v*/proof, but this one probably needs.